cmake/external/onnx_protobuf.natvis

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+  <!--Protobuf classes. These feature protobuf internals that change from version to version
```

Comment:
nit: Shall we capture this someplace else too? To make it easier for someone who is updating protobuf or onnx commit to know that they MAY need to update these classes?